pkg/agent/agent.go

```diff
@@ -119,7 +117,7 @@ func (i *Initializer) setupOVSBridge() error {
 	}
 
 	// Setup Tunnel port on OVS
-	if err := i.setupTunnelInterface(TunPortName); err != nil {
+	if err := i.setupTunnelInterface(types.DefaultTunPortName); err != nil {
```

Comment:
Do we still need to create this default tunnel port when the client wants IPSec tunnel?

Reply:
Later we need to support mode changes. I feel it would be easier to always create the default tunnel port to support changing between IPSec and not.

Reply:
Thought over. Changed to no more create tun0 for the IPSec case.